decidim-assemblies/app/permissions/decidim/assemblies/permissions.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Assemblies
+    class Permissions
+      def self.allows?(user, action, context = {})
```

Comment:
Not entirely sure this should be part of the public API. It doesn't add value to simply using `new(whatever).allowed?` and puts more strain to the implementers.

Reply:
This can be extracted into the common interface, for example, and helps with testing, as you'll only need to mock one method instead of having to create a double.

Can be removed, though. I'm not really attached to it.

Reply:
I'm ok with extracting this into the common interface, but I ain't seein' no interface. Maybe we should start with it even in the context of this PR?

Reply:
I've removed this class method.